iocore/net/QUICNetVConnection.cc

```diff
@@ -1577,14 +1570,18 @@ QUICNetVConnection::_packetize_frames(QUICEncryptionLevel level, uint64_t max_pa
           }
         }
 
-        if (!ack_eliciting && frame->type() != QUICFrameType::ACK) {
+        if (!ack_eliciting && frame->type() != QUICFrameType::PADDING && frame->type() != QUICFrameType::ACK) {
           ack_eliciting = true;
         }
 
         if (frame->type() == QUICFrameType::PING) {
           has_ping = true;
         }
 
+        if (frame->type() == QUICFrameType::PADDING) {
+          this->_padder->cancel(level);
```

Comment:
Why do we need to call `cancel()`?  It seems like `_need_to_fire` is reset to 0 in `generate_frame`.

Reply:
removed.